create-i18n/index.test.ts

```diff
@@ -233,4 +236,237 @@ test('tracks double definition', () => {
   }, /I18n component double was defined multiple times/)
 })
 
+/* 
```

Comment:
Maybe it is better to move tests to separated file like `create-i18n/partial.test.ts`

Reply:
done